doc/devel/min_dep_policy.rst

```diff
@@ -154,3 +155,28 @@ Matplotlib  Python    NumPy
 .. _`1.5`: https://matplotlib.org/1.5.0/users/installing.html#required-dependencies
 .. _`1.4`: https://matplotlib.org/1.4.0/users/installing.html#required-dependencies
 .. _`1.3`: https://matplotlib.org/1.3.0/users/installing.html#build-requirements
+
+
+Updating Python and Numpy versions
+==================================
+
+To update the minimum versions of Python we need to update:
+
+- ``pyproject.toml`` (classifiers and requires-python)
+- ``environment.yml``
```

Comment:
Also the `envlist` in `tox.ini` and the reference to it in `doc/devel/testing.rst`.